File: hbase_storage/src/main/java/com/sap/test/TestConsumer.scala
package com.sap.test

import com.sap.i076326.kafka_consumer

object TestConsumer{
  def main(args: Array[String]): Unit = {
    val consumer = new kafka_consumer("localhost:2182", "group2", "kafkaScala")
    consumer.run()

//    Thread.sleep(60000)
//    consumer.shutdown()
  }
}

File: kafka_producer/src/main/java/com/sap/test/TestConsumer.java
package com.sap.test;

import com.sap.i076326.kafka_consumer_group;

public class TestConsumer {
    public static void main(String [] args){
        kafka_consumer_group cg = new kafka_consumer_group("localhost:2181", "group1", "testTopic");
        cg.run(4);
        try{
            Thread.sleep(60000);
        }catch(InterruptedException e){

        }
        cg.shutdown();
    }
}